pkg/operator/suite_test.go

```diff
@@ -0,0 +1,42 @@
+package operator
+
+import (
+	"path/filepath"
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"k8s.io/client-go/rest"
+	"sigs.k8s.io/controller-runtime/pkg/envtest"
+	"sigs.k8s.io/controller-runtime/pkg/envtest/printer"
+)
+
+var (
+	cfg     *rest.Config
+	testEnv *envtest.Environment
+)
+
+func TestMachinesetController(t *testing.T) {
+	RegisterFailHandler(Fail)
```

Comment:
If we're using klog at all within this code, let's set the output
```suggestion
	RegisterFailHandler(Fail)
	klog.SetOutput(GinkgoWriter)
```